Branch-SDK/src/main/java/io/branch/referral/ServerRequest.java

```diff
@@ -624,6 +634,7 @@ void doFinalUpdateOnMainThread() {
         if (shouldAddDMAParams()) {
             addDMAParams();
         }
+        updateConsumerProtectionAttributionLevel();
```

Comment:
Is this function updating local storage or just setting values on the request body?

Reply:
Just updates the request. I changed the naming to addConsumerProtectionAttributionLevel